tools/performance/Invoke-PerfviewPS.ps1

```diff
@@ -0,0 +1,50 @@
+#requires -RunAsAdministrator
+
+param(
+    $etlFile = '.\perfviewdata.etl',
+
+    [Parameter(Mandatory)]
+    [scriptblock]
+    $scenario,
+    $logFile = '.\perfview.log')
+
+$scenarioEncoded = [System.Convert]::ToBase64String([System.Text.Encoding]::UNICODE.GetBytes($scenario.ToString()))
+$perfViewArgs = @(
+    '/AcceptEula'
+    '/ThreadTime'
+    "/LogFile=$logFile"
+    "/DataFile:$etlFile"
+    '/noRundown'
+    '/Merge'
+    '/Zip:False'
+#    '/ClrEvents=default+GCSampledObjectAllocationHigh'
+    '/Providers:*Microsoft-PowerShell-Runspaces,*Microsoft-PowerShell-CommandDiscovery,*Microsoft-PowerShell-Parser,*Microsoft.Windows.PowerShell'
+    'run'
+    "'$(Get-Command pwsh.exe).Source'"
```

Comment:
Can we make this a parameter? So we can launch a different pwsh.exe version than the one we are using.

Reply:
Sure.